server/server.go

```diff
@@ -1,113 +1,52 @@
 package server
 
-import (
-	"path/filepath"
-
-	pb "github.com/kubernetes/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
-	"github.com/opencontainers/ocitools/generate"
-	"golang.org/x/net/context"
+const (
+	runtimeAPIVersion = "v1alpha1"
```

Comment:
@runcom , @mrunalp , sounds good. 
Just double check I understand `runtimeAPI` correctly: doe we expect all the container runtimes have the API version?